manim/mobject/types/vectorized_mobject.py

```diff
@@ -2195,6 +2227,15 @@ def __init__(self, vmobject, **kwargs):
 class DashedVMobject(VMobject, metaclass=ConvertToOpenGL):
     """A :class:`VMobject` composed of dashes instead of lines.
 
+    Parameters
+    ----------
+        vmobject: The object that will get dashed
+        num_dashes: Number of dashes to add.
+        dashed_ratio: Ratio of dash : empty space.
+        dash_offset: Shifts the starting point of dashes along the path. Value 1 shifts 1 dash over to the next.
+        even_lengths: If true, dashes will be visually even length. If false, dashes will be split evenly in the curve's
+            input t variable (legacy behavior).
```

Comment:
Minor improvements to language and formatting. Also, I'd suggest renaming `even_lengths` to `equal_lengths`, that sounds more correct to me. Alternatively, `even_spacing` would also work.

```suggestion
        vmobject
            The object that will get dashed
        num_dashes
            Number of dashes to add.
        dashed_ratio
            Ratio of dash to empty space.
        dash_offset
            Shifts the starting point of dashes along the
            path. Value 1 shifts by one full dash length.
        even_lengths
            If ``True``, dashes will be (approximately) equally long.
            If ``False``, dashes will be split evenly in the curve's
            input t variable (legacy behavior).
```